tests/migration_test.go

```diff
@@ -3632,7 +3632,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 
 	})
 
-	Context("[Serial][QUARANTINE][test_id:8482] Migration Metrics", func() {
+	Context("[Serial][test_id:8482] Migration Metrics", func() {
```

Comment:
This needs to be reverted before we merge.

Reply:
Yup, that's right :+1: 
I just wanted to make sure the tests pass. I'll re-push the PR.